Add monitor terms to glossary (#21173)
DataDog/documentation: content/en/glossary/terms/threshold_alert.md

---
title: threshold alert
core_product:
    - alerts
related_terms:
    - change alert
---

> Monitor detection method that compares metric values to a static threshold. This is automatically selected as the default detection method for metric monitors. For more information, <a href="/monitors/types/metric/?tab=threshold#choose-the-detection-method">see the documentation</a>.

Reviewer: ```suggestion
Monitor detection method that compares metric values to a static threshold. Threshold alerts are the default detection method for metric monitors. For more information, <a href="/monitors/types/metric/?tab=threshold#choose-the-detection-method">see the documentation</a>.
```
